src/xERC20/contracts/XUltraLRT.sol

```diff
@@ -636,12 +643,16 @@ contract XUltraLRT is
     }
 
     function _increaseCrossChainTransferLimit(uint256 _limitInc) internal {
-        if ((crossChainTransferLimit + _limitInc) > totalSupply()) revert XErrors.InvalidTransferLimit();
+        uint256 oldLimit = crossChainTransferLimit;
+        if ((oldLimit + _limitInc) > totalSupply()) revert XErrors.InvalidTransferLimit();
         crossChainTransferLimit += _limitInc;
+        emit CrossChainTransferLimitChanged(msg.sender, oldLimit, crossChainTransferLimit);
```

Comment:
I feel like this will polute the space and cost a bunch of gas? do we really need to track everytime a user increases or decreases limit?

Reply:
will cost around 1000 gas max, should be fine.